ietf/templates/doc/document_draft.html

```diff
@@ -611,9 +611,9 @@
         </table>
         <div class="buttonlist">
             <a class="btn btn-primary btn-sm"
-               href="mailto:{{ doc.name }}@ietf.org?subject=Mail%20regarding%20{{ doc.name }}">
-                <i class="bi bi-envelope">
-                </i>
+               href="mailto:{{ doc_name }}@ietf.org?subject=Mail%20regarding%20{{ doc.name }}">
+               <i class="bi bi-envelope">
+               </i>
```

Comment:
This should be a full revert - there should be no changes to document_draft.html for this PR.